hedera-node/hedera-file-service-impl/src/main/java/com/hedera/node/app/service/file/impl/schemas/V0490FileSchema.java

```diff
@@ -89,7 +90,7 @@ public class V0490FileSchema extends Schema {
      * The default throttle definitions resource. Used as the ultimate fallback if the configured file and resource is
      * not found.
      */
-    private static final String DEFAULT_THROTTLES_RESOURCE = "/genesis/throttles.json";
+    private static final String DEFAULT_THROTTLES_RESOURCE = "genesis/throttles.json";
```

Comment:
Reverts change from #18922 - for consistency the `/` is prepended in `loadResourceInPackage(...)`.